### PR TITLE
JAMES-2855 upgrade diverses http related libs

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Comment by chibenwa** [comment]
Can we do the 'spark-java' dependency and  'logbook' dependency upgrade in a single PT too?

I'm too scared about past cross-dependencies issues to address them separately, to be honset...

**Comment by Arsnael** [comment]
Duc did it here for spark : https://github.com/linagora/james-project/pull/2578/commits/81ad5b1fbdb6d72b3da31687c4056de858b0e6fc

But I will do a pick :)

**Comment by Arsnael** [comment]
test this please

**Comment by Arsnael** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -651,7 +650,7 @@
         <javax.activation.groupId>javax.activation</javax.activation.groupId>
         <javax.activation.artifactId>activation</javax.activation.artifactId>
         <javax.persistence.version>1.0.2</javax.persistence.version>
-        <jaxb.version>2.2.11</jaxb.version>
+        <jaxb.version>2.3.1</jaxb.version>
```
Comment (blackheaven) [inline pom.xml]: is it used elsewhere?

Comment (Arsnael) [reply pom.xml]: seems in `mailet/mailetcontainer-camel` and `protocols/jwt`

Comment (Arsnael) [reply pom.xml]: Maybe it's not really used anymore by those modules... let me check

Comment (Arsnael) [reply pom.xml]: swagger is using jaxb as well it seems, I prefer to not touch it

**Pull request closed without merging**